internal/resolver/passthrough/passthrough.go

```diff
@@ -20,13 +20,20 @@
 // name without scheme back to gRPC as resolved address.
 package passthrough
 
-import "google.golang.org/grpc/resolver"
+import (
+	"errors"
+
+	"google.golang.org/grpc/resolver"
+)
 
 const scheme = "passthrough"
 
 type passthroughBuilder struct{}
 
 func (*passthroughBuilder) Build(target resolver.Target, cc resolver.ClientConn, opts resolver.BuildOptions) (resolver.Resolver, error) {
+	if target.Endpoint == "" && opts.Dialer == nil {
+		return nil, errors.New("passthrough resolver: missing address")
```

Comment:
`address` here can lead to confusion because an `address` in this context usually refers to addresses returned by the resolver. Could you please change the error message to be `passthrough: received empty target in Build()`.